src/test/java/org/folio/rest/impl/CurrentFiscalYearTest.java

```diff
@@ -0,0 +1,112 @@
+package org.folio.rest.impl;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+import static javax.ws.rs.core.Response.Status.NOT_FOUND;
+import static javax.ws.rs.core.Response.Status.OK;
+import static org.folio.rest.util.MockServer.addMockEntry;
+import static org.folio.rest.util.TestEntities.FISCAL_YEAR;
+import static org.folio.rest.util.TestEntities.LEDGER;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import org.folio.rest.jaxrs.model.FiscalYear;
+import org.folio.rest.jaxrs.model.Ledger;
+import org.folio.rest.jaxrs.resource.FinanceLedgers;
+import org.folio.rest.util.HelperUtils;
+import org.junit.jupiter.api.Test;
+
+import java.util.Date;
+import java.util.UUID;
+
+
+public class CurrentFiscalYearTest extends ApiTestBase {
```

Comment:
There is a test class for FiscalYear, please move all these tests to the same class

Reply:
Thanks @KVupp. I will do that. But I had no chance to do this earlier because FiscalYearTests has exist not more than 1 hour. :-)

Reply:
yea, ofcourse. Just wanted to mention about the new class in case you haven't noticed it. Thanks!